.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "no-unused-vars": "off",
```

Comment:
Instead of editing the file, you can use the inline comment or Run `npx eslint`
Take note of this in the next project.